RecursiveExtractor/Extractors/SevenZipExtractor.cs

```diff
@@ -90,9 +125,42 @@ public IEnumerable<FileEntry> Extract(FileEntry fileEntry, ExtractorOptions opti
             {
                 Logger.Debug(Extractor.DEBUG_STRING, ArchiveFileType.P7ZIP, fileEntry.FullPath, string.Empty, e.GetType());
             }
+            var needsPassword = false;
```

Comment:
Same comment -avoid duplication of same code essentially somehow to better code maintenance.

Reply:
Done.